frontend/packages/console-dynamic-plugin-sdk/src/extensions/dashboard-types.ts

```diff
@@ -80,3 +81,5 @@ export type StatusPopupSectionProps = {
   firstColumn: React.ReactNode;
   secondColumn?: React.ReactNode;
 };
+
+export type ClusterOverviewDetailsItemProps = DetailItemProps;
```

Comment:
lets not reference internal types which can easily be changed & introduce api breaks
```suggestion
export type ClusterOverviewDetailsItemProps = {
title: string;
  children: React.ReactNode;
  isLoading?: boolean;
  error?: boolean;
  valueClassName?: string;
  errorMessage?: string;
};
```

Reply:
done